pkg/environment/env.go

```diff
@@ -99,38 +96,19 @@ func InitEnv(options repository.Options) (*Environment, error) {
 		return nil, err
 	}
 
-	var repo *repository.Repository
 	var v1repo *repository.V1Repository
```

Comment:
should we rename `v1repo` to `repo`? And likewise `v1Repo` and `V1Repository`

Reply:
and V1Manifest, etc.

Reply:
I think it's OK, as we don't guarantee there won't be `v2repo` in the future. The only misleading thing might be that we still use `v0.1.0` as `spec_version` in `v1Manifest`.